src/styles/blocks/expo.scss

```diff
@@ -0,0 +1,115 @@
+.expo {
+  &__photo-container {
+    position: relative;
+
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 420px;
+    margin-bottom: 20px;
+    overflow: hidden;
+
+    @include onTablet {
+      margin-bottom: 30px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 20px;
+    }
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include onTablet {
+      max-height: 484px;
+      padding-bottom: 0;
+    }
+
+    @include onDesktop {
+      height: 484px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+    transition: all 0.5s;
+
+    &:hover {
+      scale: 1.05;
+    }
+  }
+
+  &__description {
+    overflow: hidden;
+
+    @include onTablet {
+      min-width: 330px;
+    }
+
+    @include onDesktop {
+      max-width: 650px;
+    }
+  }
+
+  &__button {
+    margin-bottom: 20px;
+
+    &:first-child {
+      margin-bottom: 0;
+    }
+
+    @include onTablet {
+      margin-bottom: 0;
+    }
+
+    @include onDesktop {
+      width: 50%;
+    }
+  }
```

Comment:
> please look at section 1 (exhibition), or rather at its description. I think this is hardcode, but I have no idea how to make it easier to make the "buy a ticket" buttons be at the same level when the window is stretched starting from a width of 768px

Do you mean this part of the code?
it looks like you need `margin-bottom` only in mobile view

In this case, you can move 66 row to `onDesktop` and delete `first:child` selector

![image](https://user-images.githubusercontent.com/86923089/195163636-e2e93804-552e-4c3d-9353-3ab006fab028.png)